libs/net/elastic/Models/Converters/AggregationsConverter.cs

```diff
@@ -50,13 +50,16 @@ private static AggregationRootModel ParseAggregationRootModels(ref Utf8JsonReade
             switch (propName.ToLower())
             {
                 case var _ when propName.Equals(PropNameDocCount.ToLower()):
-                    dto.DocCount = reader.GetInt64();
+                    dto.DocCount = reader.GetInt64(); // this result is not reliable
                     break;
                 default:
                     dto.ChildAggregation = ParseAggregationModel(ref reader, propName);
                     break;
             }
         }
+        // calculate the sum DocCount of ChildAggregation.Buckets
+        if (dto.ChildAggregation.Buckets.Any())
+            dto.DocCount = dto.ChildAggregation.Buckets.Sum(b => b.DocCount);
```

Comment:
if there are child aggregations manually calculate the same ignoring the doc count from elastic.  Elastic doesn't exclude parent items when child is excluded